src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/ia/CftQueryServiceITTest.java

```diff
@@ -975,7 +975,8 @@ private static Stream<TaskQueryScenario> grantTypeWithAvailableTasksOnlyOnReques
             new SearchParameterList(JURISDICTION, SearchOperator.IN, List.of(IA_JURISDICTION)),
             new SearchParameterList(LOCATION, SearchOperator.IN, List.of("765324")),
             new SearchParameterRequestContext(REQUEST_CONTEXT, SearchOperator.BOOLEAN,
-                RequestContext.AVAILABLE_TASK_ONLY)
+                RequestContext.AVAILABLE_TASKS
```

Comment:
I think only the new field should be changed i.e. `request_context`.  The existing one which is a boolean needs to stay the same for backward compatibility.  Can we check this please?